### ISSUE TITLE
Child Process Error DocFX VSCode Extension

### ISSUE THREAD

**Issue by sahammer** [dotnet/docfx#2612]
**DocFX Version Used**: 0.1.1

**Template used**: Custom [Microsoft Docs Extension](https://docs.microsoft.com/en-us/teamblog/docs-extension)

**Steps to Reproduce**:

1. Create a markdown file
2. Press Alt-M and select preview
3. [Child process Error]: 'C:\Users\Sterling' is not recognized as an internal or external command,
operable program or batch file.

**Expected Behavior**: A preview should open

**Actual Behavior**: An error is displayed

This is basically #2076  - I have also experienced this error, it is specifically with the [DocFX](https://github.com/dotnet/docfx/tree/dev/tools/VscExtension) extension that in installed with the new Docs Markdown extension:

https://docs.microsoft.com/en-us/teamblog/docs-extension

I believe this is caused by a user profile that has a space in it. My profile path is currently _C:\Users\Sterling Hammer_ This appears to be an ongoing issue.

**Comment by sahammer** [comment]
Additional context,

I believe the issue is here:

docfx/tools/VscExtension/src/Proxy/requestProxy.ts

With the following:

```
 let exePath = context.asAbsolutePath("./DfmHttpService/DfmHttpService.exe");
```

**Comment by 928PJY** [comment]
hi @sahammer, sorry for the late response, please take a look at this [issue](#2383),
we have fixed this issue in version-0.1.2, please update your extension
